### DISCUSSION TITLE
Is there any limit on the number of partitions in a partitioned db ?

### DISCUSSION THREAD

**Discussion by yhilem** [apache/couchdb#3876] [General] [answered]
Hi,
I read that "documents belonging to the same partition reside on the same shard".
Is there a limit on the number of partitions in a partitioned db ?
I have a use case where the partition key has 4K different values.
Thanks
Youcef HILEM

**Comment by kocolosk** [comment, marked as answer]
Nope, create as many as you like, up to the scalability limits of a regular CouchDB database. In many respects an un-partitioned DB behaves like a partitioned DB where every document is in its own unique partition.
